### PR TITLE
docs(ingest): Rename csv / s3 / file source and sink

### PR DESCRIPTION
**Pull request by asikowitz**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: docs-website/build.gradle [hunk 1]
```diff
@@ -148,8 +148,12 @@ clean {
   delete 'tmp'
   delete 'build'
   delete 'just'
+  delete 'sphinx/venv'
+  delete 'sphinx/_build'
+  delete 'versioned_docs'
   delete fileTree(dir: 'genDocs', exclude: '.gitignore')
   delete fileTree(dir: 'docs', exclude: '.gitignore')
+  delete fileTree(dir: 'genStatic', exclude: '.gitignore')
```
Comment (asikowitz) [inline docs-website/build.gradle:156]: Deleting files that are not persisted by git

File: .gitignore [hunk 2]
```diff
@@ -126,3 +126,6 @@ metadata-service/war/bin/
 metadata-utils/bin/
 test-models/bin/
 
+datahub-executor/
+datahub-integrations-service/
+metadata-ingestion-modules/
```
Comment (asikowitz) [inline .gitignore:131]: Think this is fine to include? Tired of seeing these directories when switching repos

File: metadata-ingestion/setup.py [hunk 3]
```diff
@@ -259,7 +259,8 @@
 
 delta_lake = {
     *s3_base,
-    "deltalake>=0.6.3, != 0.6.4",
+    "deltalake>=0.6.3, != 0.6.4, < 0.18.0; platform_system == 'Darwin' and platform_machine == 'arm64'",
+    "deltalake>=0.6.3, != 0.6.4; platform_system != 'Darwin' or platform_machine != 'arm64'",
```
Comment (asikowitz) [inline metadata-ingestion/setup.py:263]: Seems like 0.18.0 is broken on ARM macs. Can alternately just forbid version 0.18.0 but may run into the same issue on the next version

**Review by asikowitz — COMMENTED** [review]
Some somewhat significant changes, would appreciate a deep review on the non-md changes

File: .gitignore (hunk 2 above)
Comment (hsheth2) [reply .gitignore:131]: should be `metadata-ingestion-modules/acryl-cloud`

File: metadata-ingestion/setup.py (hunk 3 above)
Comment (hsheth2) [reply metadata-ingestion/setup.py:263]: can we also link to the issue https://github.com/delta-io/delta-rs/issues/2577

**Review by hsheth2 — APPROVED** [review]
Thanks for cleaning this up

**Pull request merged**